templates/static/form_handler.js

```diff
@@ -149,3 +171,50 @@ document.getElementById('url-form').addEventListener('submit', async function(ev
         }, 'image/png');
     });
 });
+
+async function downloadImage(imageElement, text, headlineElement, fontSize, color, fontType) {
```

Comment:
Remove `downloadImage` from `templates/static/form_handler.js`.
Use the one in `templates/static/image_manipulation.js`

Reply:
Thank you for pointing that out. I will remove the `downloadImage` function from `templates/static/form_handler.js` and ensure the one in `templates/static/image_manipulation.js` is used correctly. I'll push the fix shortly.